### PR TITLE
Separate dask job

### PR DESCRIPTION
**Pull request by chukarsten**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,23 +19,35 @@ lint-fix:
 
 .PHONY: test
 test:
-	pytest evalml/ --doctest-modules --doctest-continue-on-failure
+	pytest evalml/ --doctest-modules --doctest-continue-on-failure --ignore=evalml/tests/automl_tests/dask_tests
+
+.PHONY: test-dask
+test:
+	pytest evalml/tests/automl_tests/dask_tests/ --doctest-modules --doctest-continue-on-failure
```
Comment (freddyaboulton) [inline Makefile:26]: Can you add the `--timeout 360` ?

File: Makefile [hunk 2]
```diff
@@ -19,23 +19,35 @@ lint-fix:
 
 .PHONY: test
 test:
-	pytest evalml/ --doctest-modules --doctest-continue-on-failure
+	pytest evalml/ --doctest-modules --doctest-continue-on-failure --ignore=evalml/tests/automl_tests/dask_tests
```
Comment (freddyaboulton) [inline Makefile:22]: I'd rather we still had one command to run all tests but this is not blocking merge.

File: .github/workflows/windows_unit_tests_with_dask.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+
+jobs:
+  win_unit_tests:
```
Comment (freddyaboulton) [inline .github/workflows/windows_unit_tests_with_dask.yml:11]: I'll file an issue for trying to reduce as many of the set-up steps as possible between the dask-only and non-dask tests. I think that'll pay off in the future.

**Review by freddyaboulton — APPROVED** [review]
Thanks @chukarsten !

File: Makefile (hunk 2 above)
Comment (chukarsten) [reply Makefile:22]: I'll do a quick PR to reintroduce this.  Good point

**Pull request merged**